Whiskey/Tasks/Invoke-WhiskeyPipelineTask.ps1

```diff
@@ -0,0 +1,105 @@
+ function Invoke-PipelineTask 

```

Comment:
Oh, hah! I just noticed the typo. I didn't look at the code. Yes. This should be named `Invoke-WhiskeyPipelineTask`.